scalableminds/webknossos
# List Currently Available Mappings

### URL of deployed dev instance (used for testing):
- https://listmappings.webknossos.xyz

### Steps to test:
- you need a dataset with mappings
- back end test (probably covered by front end testing): check `http://localhost:9000/data/datasets/Connectomics_Department/ROI2017_wkw/layers/segmentation/mappings?token=secretScmBoyToken`
- front end test:
  - open a dataset with mappings (view or skeleton mode)
  - enable mappings in the segmentation tab and select a mapping
  - everything should work as you expect

### Screenshot & Gif

![image](https://user-images.githubusercontent.com/2486553/52468084-e68b9f00-2b87-11e9-8cd1-b9f3aa944dce.png)

![mapping-ui](https://user-images.githubusercontent.com/2486553/52468250-72053000-2b88-11e9-9042-1130a2e45485.gif)


### Issues:
- fixes #3648 
- fixes #3501 

------
